### ISSUE TITLE
Unable to launch the tool

### ISSUE THREAD

**Issue by Observer365** [lostindark/DriverStoreExplorer#158]
I've tried safe mode, .net installations, visual studio installations and force launching the app

The .exe seems to crash/killed everytime i open it

I realize this could be an issue on my machine only, is there any place that i can start looking? Do you have any info on this issue?

**Comment by lostindark** [comment]
Try check system event log. It should tell you why the app crashed.

**Comment by Capricus02** [comment]
Just want to say that I'm having this issue as well. Looking at the event log, it appears to be an unauthorized access issue, however, I'm running the program as administrator (both logged in and right-click > Run as Administrator) and it still will not load / automatically crashes after start up? Here is one of the event logs:

Log Name:      Application
Source:        .NET Runtime
Date:          3/18/2021 3:13:32 PM
Event ID:      1026
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      MSI
Description:
Application: Rapr.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.UnauthorizedAccessException
   at System.IO.__Error.WinIOError(Int32, System.String)
   at System.IO.FileSystemEnumerableIterator`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].AddSearchableDirsToStack(SearchData)
   at System.IO.FileSystemEnumerableIterator`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].MoveNext()
   at Rapr.DSEFormHelper.GetSupportedLanguage()
   at Rapr.DSEForm..cctor()

Exception Info: System.TypeInitializationException
   at Rapr.DSEForm.BuildLanguageMenu()
   at Rapr.DSEForm..ctor()
   at Rapr.Program.Main()


Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name=".NET Runtime" />
    <EventID Qualifiers="0">1026</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>0</Task>
    <Opcode>0</Opcode>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2021-03-18T19:13:32.3593509Z" />
    <EventRecordID>505</EventRecordID>
    <Correlation />
    <Execution ProcessID="0" ThreadID="0" />
    <Channel>Application</Channel>
    <Computer>MSI</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Application: Rapr.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.UnauthorizedAccessException
   at System.IO.__Error.WinIOError(Int32, System.String)
   at System.IO.FileSystemEnumerableIterator`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].AddSearchableDirsToStack(SearchData)
   at System.IO.FileSystemEnumerableIterator`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].MoveNext()
   at Rapr.DSEFormHelper.GetSupportedLanguage()
   at Rapr.DSEForm..cctor()

Exception Info: System.TypeInitializationException
   at Rapr.DSEForm.BuildLanguageMenu()
   at Rapr.DSEForm..ctor()
   at Rapr.Program.Main()

</Data>
  </EventData>
</Event>

**Comment by Observer365** [comment]
I got this solved after realizing it was blocked by my OEM. I have disabled updates for bios but somehow a new bios update managed to get through along with Asus's bloatware updates. I had to delete all drivers installed to get the rapr.exe to launch. 
Although it did launch, it did not detect drivers, I did a system restore, blocked all updates and reinstalled necessary software and older bios and rapr.exe launched fine this time. 

found additional weird .inf files from asus and deleted them. 

I don't wanna say it but maybe my OEM ie: Asus, is blocking this software so they can continue installing updated telemetry and data gathering utilities to gather data by circumventing latest store/site needs. A good example of this would be the recent stable releases of armoury crate being moved to Microsoft Store beta to avoid detection.
